ansible/roles/ocp4-workload-workshop-admin-storage/tasks/pre_workload.yml

```diff
@@ -1,6 +1,41 @@
 ---
 # Implement your Pre Workload deployment tasks here
 
+- name: Cleaning admin
+  include_tasks:
+    file: ./clean-admin.yml
+
+- name: extract the sandbox id
+  shell: "echo {{ target_host }} | cut -d. -f3"
+  register: sandbox_id
```

Comment:
I don't see where you use `sandbox_id` in the rest of the workload. Maybe you can remove that part ?
Also, what happen if the workload is not loaded in a sandbox account but in a standard account ?


We can pass the `sandbox_account` to agnosticd when deploying, that way you won't have to guess is using `target_host` or dns names.

If you tell me which catalog item is using this we can make sure `sandbox_account` is defined.

Reply:
This one
![image](https://user-images.githubusercontent.com/11228024/60920797-0c1bf200-a267-11e9-8ec6-45ee398092db.png)